inheritance/src/main/java/inheritance/Theater.java

```diff
@@ -0,0 +1,54 @@
+package inheritance;
+
+import java.util.LinkedList;
+
+public class Theater implements Reviewable {
+    String name;
+    LinkedList<String> movies = new LinkedList<>();
+    LinkedList<Review> reviews = new LinkedList<>();
+
+    public Theater(String name, LinkedList<String> movies) {
+        this.name = name;
+        this.movies = movies;
+    }
+
+    public void addMovie(String movie) {
+        movies.add(movie);

```

Comment:
Food for thought: Is there a way to prevent users from adding the same movie more than once?